src/Form/FormControl.js

```diff
@@ -11,6 +11,7 @@ export const styleSheet = createStyleSheet('FormControl', () => {
       display: 'flex',
       flexDirection: 'column',
       position: 'relative',
+      transform: 'translateZ(0)', // #6076 prevents animation glitches of children to bubble up
```

Comment:
On the other hand, users should have many Textfield component on their page.
Maybe we can accept that tradeoff. It's not like https://github.com/callemall/material-ui/pull/5868.